source/documentation/breaking-changes/moz-document.md.erb

```diff
@@ -0,0 +1,37 @@
+---
+title: "Breaking Change: -moz-document"
+introduction: >
+  Firefox used to have a @-moz-document rule requiring a special parsing. As
+  support is removed from Firefox, Sass is in the process of removing support
+  for parsing them.
+---
+
+Sass has historically supported a special parsing for the `@-moz-document` rule.
+As [Firefox dropped support for them], Sass will also drop support.
```

Comment:
Per this comment - https://github.com/sass/dart-sass/issues/378#issuecomment-1238704220
Sass won't drop support entirely, only for the special parsing in favor of interpolation within unknown at-rules